groonga-release/Rakefile

```diff
@@ -23,6 +23,13 @@ class GroongaReleasePackageTask < PackagesGroongaOrgPackageTask
     "RPM-GPG-KEY-#{shorten_gpg_key_id(id).downcase}"
   end
 
+  def repository_gpg_key_ids
+    super.select do |gpg_key_id|
+      # This GPG key has low strength, so we will no longer use it.
+      gpg_key_id != "C97E4649A2051D0CEA1A73F972A7496B45499429"
```

Comment:
Can we remove this from https://github.com/groonga/groonga/blob/0d9961cc1a146b4ddb6e9341e2274c2c772c89a6/packages/repository-helper.rb#L31 ?